examples/coappearances.cpp

```diff
@@ -13,6 +13,20 @@
 
 namespace co = coappearances;
 
+void
+convert_meta( const picojson::object& obj, co::GraphBuilder builder, std::string& strings )
+{
+    // Since flatdata's mutators are not holding any data, we are creating a vector with a single
+    // element for holding the data.
+    flatdata::Vector< co::Meta > data( 1 );
```

Comment:
Is there a better way to do it right now? -- If not, we should think about a better approach. I find this very contra-intuitive.

One approach could be introducing a method `start_meta` to the builder, which would hide the vector with a single field from the user. `close`  method would then do what `set_meta` is doing now.

Reply:
We always wanted to add a `flatdata::Struct< T >` or `flatdata::Object< T >` but there was no time (aka good reason) to. Could do that, if you like (:

Reply:
Ok, referenced in #23.